lexoid/api.py

```diff
@@ -213,6 +214,30 @@ def parse(
                     "total": sum(r["token_usage"]["total"] for r in chunk_results),
                 },
             }
+
+            if "api_cost_mapping" in kwargs:
+                api_cost_mapping = kwargs["api_cost_mapping"]
+                if isinstance(api_cost_mapping, dict):
+                    api_cost_mapping = api_cost_mapping
+                elif isinstance(api_cost_mapping, str) and os.path.exists(
+                    api_cost_mapping
+                ):
+                    with open(api_cost_mapping, "r") as f:
+                        api_cost_mapping = json.load(f)
+                else:
+                    raise ValueError(f"Unsupported API cost value: {api_cost_mapping}.")
+
+            price_per_m = api_cost_mapping.get(
+                kwargs.get("model", "gemini-2.0-flash"), None
+            )
+            if price_per_m:
+                token_cost = {
+                    "input": result["token_usage"]["input"] * price_per_m["input"],
```

Comment:
We might need to adjust the price for per 1M token?
Something like,
```
  token_cost = {
      "input": float(
          result["token_usage"]["input"]
          * price_per_m["input"]
          / 1000000
      ),
      "output": float(
          result["token_usage"]["output"]
          * price_per_m["output"]
          / 1_000_000
      ),
  }
```

Reply:
Fixed.